packages/frontend/src/ee/features/aiCopilot/components/ChatElements/AiChartVisualization.tsx

```diff
@@ -215,12 +216,16 @@ export const AiChartVisualization: FC<Props> = ({
                                         />
                                     )}
 
-                                {message.filtersOutput && (
+                                {message.vizConfigOutput &&
+                                'filters' in message.vizConfigOutput &&
+                                message.vizConfigOutput.filters ? (
                                     <AgentVisualizationFilters
                                         // TODO: fix this using schema
-                                        filters={message.filtersOutput}
+                                        filters={filtersSchemaTransformed.parse(
+                                            message.vizConfigOutput.filters,
+                                        )}
                                     />
-                                )}
+                                ) : null}
```

Comment:
I think there's an easier fix for this:

https://app.graphite.dev/github/pr/lightdash/lightdash/15643